react/src/components/buttons/SprkButton.test.js

```diff
@@ -159,6 +237,18 @@ describe('SprkButton:', () => {
     },
   );
 
+  it(
+    'it should apply the disabled attribute if the element is a button' +
+      ' and disabled is true',
+    () => {
+      const wrapper = mount(<SprkButton isDisabled="true" />);
+      const button = wrapper.find('button');
+      expect(button.length).toBe(1);
+      expect(button.getDOMNode().hasAttribute('disabled')).toBe(true);
+    },
+  );
+
+  // TODO: Deprecate
```

Comment:
Looks great!

Can we add a couple new tests to verify that the new Props are preferred over the deprecated Props?
```
<SprkButton isSpinning={true} loading={false} />

<SprkButton isDisabled={true} disabled={false} />
```

Reply:
After our conversation a few minutes ago, the way booleans work, it's either there or it's not. If it's set to false, it's evaluated as not there at all.
There's no way to prefer the new prop over the old one because of the mentioned situation.

We're skipping writing these tests for now.

Reply:
the suggestion i made in there should set it up so that the new one is preferred